.travis.yml

```diff
@@ -1,9 +1,22 @@
 sudo: required
+cache: bundler
+language: ruby
+
 services:
   - docker
 
 install:
-  - export BUNDLE_GEMFILE=$PWD/Gemfile
   - bundle install
 
-script: bundle exec kitchen verify ${INSTANCE}
+env:
+  matrix:
+    - INSTANCE: centos-7
+    - INSTANCE: debian-9
+    - INSTANCE: ubuntu-1804
```

Comment:
How about introducing the larger matrix we're working towards, i.e. https://github.com/saltstack-formulas/template-formula/pull/38?

Perhaps best to leave this for another a PR...